src/backend/chat_service/chat_service.py

```diff
@@ -61,12 +63,33 @@ async def add_correlation_id(request: Request, call_next):
 async def add_customer(request: Request):
     logger.debug(f"Entering add_customer() with Correlation ID: {request.state.correlation_id}")
     customer_guid = db_manager.add_customer()
+    minio_manager=  MinioManager()
+    weaviate_manager=WeaviateManager()
+
     if customer_guid is None:
         logger.error("Failed to create customer")
         raise HTTPException(status_code=500, detail="Failed to create customer")
-    logger.debug(f"Exiting add_customer() with Correlation ID: {request.state.correlation_id}")
-    return {"customer_guid": customer_guid}
 
+    #create a MinIO bucket
+    try:
+        minio_status=minio_manager.add_storage_bucket(customer_guid)
+    except Exception as e:
+        logger.error(f"Error creating MinIO bucket: {e}")
+        raise HTTPException(status_code=500, detail=f"Error creating MinIO bucket: {e}")
```

Comment:
The reply to this API should be the same  raise HTTPException(status_code=500, detail="Failed to create customer") Dont give details like MInio bucket creation error

Make the logger with the correlation id
logger.error(f"Error creating MinIO bucket: {e}") The logging message we can give that Minio bucket creation failed